apps/src/codebridge/FileBrowser/types.ts

```diff
@@ -4,16 +4,8 @@ import {ProjectFileType} from '@cdo/apps/lab2/types';
 
 export type downloadFileType = (fileId: FileId) => void;
 export type moveFilePromptType = (fileId: FileId) => void;
-export type moveFileType = (
-  fileId: FileId,
-  destinationFolderName: string
-) => void;
 export type newFilePromptType = (folderId?: FolderId) => void;
-export type newFileType = (folderId: FolderId, fileName: string) => void;
 export type newFolderPromptType = (parentId?: FolderId) => void;
-export type newFolderType = (parentId: FolderId, folderName: string) => void;
 export type renameFilePromptType = (fileId: FileId) => void;
-export type renameFileType = (fileId: FileId, newName: string) => void;
 export type renameFolderPromptType = (folderId: FolderId) => void;
-export type renameFolderType = (folderId: FolderId, newName: string) => void;
 export type setFileType = (fileId: FileId, type: ProjectFileType) => void;
```

Comment:
These types had only recently been added in #60192, and it was strictly to allow the dialogs to have callbacks that wake up upon `confirm`. Since we can now invoke the dialog asynchronously, we can nuke these things + their support functions